platform/qt/src/qmapboxgl_p.hpp

```diff
@@ -37,7 +37,7 @@ class QMapboxGLPrivate : public QObject, public mbgl::RendererFrontend
     void render();
     void setFramebufferObject(quint32 fbo, const QSize& size);
 
-    using PropertySetter = std::function<mbgl::optional<mbgl::style::conversion::Error>(mbgl::style::Layer&, const std::string&, const mbgl::style::conversion::Convertible&)>;
```

Comment:
Could you please provide more details of this change? I'm curious whether you found out that passing `std::function` is slower than a plain function pointer.

Reply:
`std::function` is slower and bigger than a function pointer. As a general rule, if you can reasonably avoid using `std::function`, it's a good idea to do so.